content/user/order_total/coupons.md

```diff
@@ -67,3 +67,8 @@ Once they fill in the code and press Enter, it will be checked for validity, and
 If they wish to remove a coupon, they can enter a different coupon code, or they can type "REMOVE", and press Enter.
 
 ![Discount Coupon Entry](/images/discount_coupon_entry.png)
+
+## Coupon Types 
+
+The `coupons` table has a `coupon_type` field. The possible values for coupon_type are shown in the `collect_posts` function in `includes/modules/order_total/ot_coupon.php` in Zen Cart 1.5.8 or higher. 
```

Comment:
Is it worth notating these sorts of notes (that are more dev-related) in a way that's easy to identify them later in order to move them to the dev section of the docs?

Reply:
I am open to that - I just wanted to get this in for now.  You're right - it's not truly a storeowner issue.  The order totals section in the dev area didn't have a file that was a great fit for this content.